core/version.h

```diff
@@ -62,18 +62,26 @@
 // Example: 3.1.4 will be 0x030104, making comparison easy from script.
 #define VERSION_HEX 0x10000 * VERSION_MAJOR + 0x100 * VERSION_MINOR + VERSION_PATCH
 
+#ifdef REAL_T_IS_DOUBLE
+#define VERSION_DOUBLE ".double"
```

Comment:
Since this file is included widely in the codebase, let's make sure it doesn't accidentally get used instead of `REAL_T_IS_DOUBLE`.
I understand the other macros have similar short names, but I'd prefer something like `BUILD_SUFFIX_REAL_T_IS_DOUBLE_PART`.

Reply:
See https://github.com/godotengine/godot/pull/99590#discussion_r2006419366, we don't need an extra define at all.

For the record the other macros were already renamed to have a `GODOT_` prefix, this PR is pending a rebase.